lib/core/src/animation/differential_time_animator.rs

```diff
@@ -0,0 +1,47 @@
+use super::ContinuousTimeAnimator;
```

Comment:
Please try to refactor all files with underscores to nested strucutres. Sometimes it is not possible, but here it is definitely possible and makes the strucutre much nicer. Here it should be moved to `src/animation/animator/differential.rs`.

Reply:
I tried it and TBH it felt weird. I think a good way to group the animators should be something like this:

```
animation/animators/animator.rs
animation/animators/continuous_animator.rs
animation/animators/fixed_step_animator.rs
animation/physics.rs
```

Side note: `differential_time_animator.rs` was pushed by mistake. As we agreed before, this file was renamed to `animator.rs`

Reply:
Ok, thanks for the explanation!
Btw, why you feel it is better this way than this way?

```
animation/animators/animator.rs
animation/animators/animator/continuous.rs
animation/animators/animator/fixed_step.rs
animation/physics.rs
```

Imo, it is much, much more readable as we keep that hierarchical. Especially as `ContinousAnimator` and `FixedStepAnimator` are kind-of animators, so `animator.rs` should export all of them.